nilearn/plotting/data/html/surface_plot_template_niivue.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+
+  <head>
+    <meta charset="utf-8">
+    <title>$INSERT_PAGE_TITLE_HERE</title>
+    <link rel="stylesheet" href="https://niivue.github.io/niivue/features/niivue.css">
+  </head>
+  <body>
+    <canvas id="gl"></canvas>
+  </body>
+  $INSERT_JS_LIBRARIES_HERE
+
+  <script type="module" async>
+    var nv = new niivue.Niivue();
+    nv.attachTo('gl');
+
+    let layers = [
+      {
+          name: "bg_map.gii",
+          useNegativeCmap: false,
+          opacity: 0.7,
+          colormap: "gray",
+          base64: "$INSERT_BG_MAP_BASE64_HERE",
+      },
+      {
+          name: "surf_map.gii",
+          useNegativeCmap: true,
+          opacity: 0.7,
+          cal_min: $INSERT_THRESHOLD_HERE,
+          base64: "$INSERT_SURF_MAP_BASE64_HERE",
+      },
+    ];
+
+    let m = await niivue.NVMesh.loadFromBase64({
+      gl: nv.gl,
+      name: "pial_left.gii",
```

Comment:
```suggestion
      name: "surf_mesh.gii",
```
Or is there a reason the name should be specifically "pial_left.gii" here?